src/screens/Survey/index.test.tsx

```diff
@@ -16,22 +23,67 @@ describe('SurveyScreen', () => {
     );
   };
 
-  it('renders Survey screen and its components', () => {
-    render(<TestComponent />);
-
-    const backgroundImage = screen.getByTestId(mainViewTestIds.base);
-    const backButton = screen.getByTestId(surveyScreenTestIds.backButton);
-    const coverImage = screen.getByTestId(surveyScreenTestIds.coverImage);
-    const title = screen.getByTestId(surveyScreenTestIds.title);
-    const description = screen.getByTestId(surveyScreenTestIds.description);
-    const startSurveyButton = screen.getByTestId(surveyScreenTestIds.startSurveyButton);
-
-    expect(backgroundImage).toBeVisible();
-    expect(backButton).toBeVisible();
-    expect(coverImage).toBeVisible();
-    expect(title).toBeVisible();
-    expect(description).toBeVisible();
-    expect(startSurveyButton).toBeVisible();
-    expect(startSurveyButton).toHaveTextContent('Start Survey');
+  const mockState: { survey: SurveyState } = {
+    survey: {
+      survey: {
+        id: faker.string.uuid(),
+        resourceType: 'survey',
+        title: faker.string.sample(),
+        description: faker.string.sample(),
+        coverImageUrl: faker.image.url(),
+      },
+      isLoading: true,
+      isError: false,
+    },
+  };
+
+  beforeEach(() => {
```

Comment:
Could you move the `beforeEach` and `afterEach` to the top of tests inside this group?

Reply:
@hoangmirs updated in [bdf9372](https://github.com/manh-t/react-survey/pull/70/commits/bdf937279167ce31777df865d25115d78e6457c8)